Решение задач блока 00 - intro и 01-basics

rainurka opened the pull request:
Добавил решение задачи sum, CounterButtonApp,VueCalculator и MarkedList

rainurka committed:
Решение задач блока 00 - intro и 01-basics

jsru commented:
Спасибо за отправленный пулриквест, теперь @ShGKme должен запустить тесты.

jsru commented:
Проверьте, пожалуйста, ваше решение, не все тесты прошли (PR не будет принят до тех пор, пока **все** добавленные задачи не будут решены).

ShGKme reviewed (commented):
Все решённые задачи приняты, но требуется решить все добавленные задачи, чтобы PR был принят.

rainurka committed:
Changes to be committed:
	modified:   00-intro/01-sum/README.md
	modified:   01-basics/02-VueCalculator/index.html
	modified:   01-basics/02-VueCalculator/script.js
	modified:   package-lock.json

 Changes not staged for commit:
	modified:   01-basics/02-VueCalculator/script.js
	modified:   01-basics/04-MeetupTitle/index.html
	modified:   01-basics/04-MeetupTitle/script.js

rainurka committed:
Changes to be committed:
	modified:   01-basics/02-VueCalculator/script.js
	modified:   01-basics/04-MeetupTitle/index.html
	modified:   01-basics/04-MeetupTitle/script.js

rainurka committed:
Merge branch 'master' of https://github.com/js-tasks-ru/vue-20220125_rainur

rainurka committed:
Merge branch 'master' of https://github.com/js-tasks-ru/vue-20220125_rainur

ShGKme commented:
👀
